### PR TITLE
prompt suggestions

### PR DESCRIPTION
**Pull request by danilopcarlotti**
I created a simple commit
The latest comment about this commit was that it was approved, the problem was that this PR had a parent that it didn't need to have

### CODE REVIEW

File: examples/prompt_examples/prompt_suggestions.py
Comment (aidylka) [inline examples/prompt_examples/prompt_suggestions.py]: typo:
```suggestion
TWEETS = "Consider these social media mentions. Do they mention [TARGET]? Are they positive or negative? Create a report with bullet points in markdown style with the following sections: Positive mentions, Neutral mentions, Negative mentions."
```

**Review by aidylka — COMMENTED** [review]
Changes LGTM. 

Can you add back the readme you had on the original PR? https://github.com/airtop-ai/examples-python/pull/3/files#diff-aafc27024ea845820852d131f5b4370e735f0c73d3587e388d748975850d1641R1

Just a small typo and a comment. I'll approve it once those are in.

**Comment by danilopcarlotti** [comment]
> Changes LGTM.
> 
> Can you add back the readme you had on the original PR? https://github.com/airtop-ai/examples-python/pull/3/files#diff-aafc27024ea845820852d131f5b4370e735f0c73d3587e388d748975850d1641R1
> 
> Just a small typo and a comment. I'll approve it once those are in.

Done

File: examples/prompt_examples/prompt_suggestions.py
Comment (aidylka) [inline examples/prompt_examples/prompt_suggestions.py]: typo
```suggestion
SENTIMENT = "What is the sentiment described on this site? Does it mention [TARGET]? Is it positive or negative? Create a report with bullet points in markdown style with the following sections: Positive mentions, Neutral mentions, Negative mentions."
```

**Review by aidylka — APPROVED** [review]
Found 1 more typo. The rest looks good! Please add that change before merging and go ahead and merge. Nice prompt suggestions on this PR!

**Pull request merged**
